src/arkit.cs

```diff
@@ -67,7 +67,9 @@ public enum ARErrorCode : long {
 		InvalidReferenceObject = 301,
 		InvalidWorldMap = 302,
 		InvalidConfiguration = 303,
+		[Deprecated (PlatformName.iOS, 13, 2, message : "Please use the 'InvalidCollaborationData' value instead.")]
 		CollaborationDataUnavailable = 304,
```

Comment:
I realize that error codes don't get their own availability attributes (https://github.com/xamarin/xamarin-macios/pull/7322/commits/36c637056af3b932341f20aad8c9b19c63aecc46#r339817547). How would we handle deprecation of the same though? Or give the suggested enum value?

Reply:
Deprecation AFAIK is correct, but I would have done the following:
```csharp
CollaborationDataUnavailable = InvalidCollaborationData,
InvalidCollaborationData = 304,
```

So that we are always sure they are the same value.